api/src/main/java/org/pmiops/workbench/db/model/DbUser.java

```diff
@@ -482,9 +482,9 @@ public String generatePDName() {
     return getUserPDNamePrefix() + "-" + randomStringForUserResource();
   }
 
-  /** Returns a name for the persistent disk used in APP to be created for this user. */
+  /** Returns a name for the persistent disk to be created for this user for a given AppType. */
   @Transient
-  public String generatePDNameForApp(AppType appType) {
+  public String generatePDNameForUserApps(AppType appType) {
```

Comment:
We could rename AppType to UserAppType, but I don't think that's necessary.  What other kind of app would have types?